src/apache/conf/httpd.conf

```diff
@@ -59,6 +60,29 @@ LoadModule dir_module modules/mod_dir.so
 LoadModule alias_module modules/mod_alias.so
 LoadModule rewrite_module modules/mod_rewrite.so
 LoadModule socache_shmcb_module modules/mod_socache_shmcb.so
+<IfDefine EnableHttp2>
+  LoadModule http2_module modules/mod_http2.so
+</IfDefine>
+
+
+
+<IfModule http2>
+  # Server Protocols
+  Protocols h2 h2c http/1.1
+
```

Comment:
Could you turn on the [H2Direct](http://wild.mat.ucm.es/manual/en/mod/mod_http2.html#h2direct) directive like they did here? https://github.com/nextcloud/vm/blob/master/nextcloud_install_production.sh#L380

Reply:
Thanks for noticing. Must have forgot that option. Is now implemented.